src/js/actions/libraries.js

```diff
@@ -464,9 +464,9 @@ define(function (require, exports) {
                 newElement = currentLibrary.createElement(currentLayer.name, ELEMENT_LAYERSTYLE_TYPE);
 
                 return Promise.fromCallback(function (done) {
-                        var representation = newElement.createRepresentation(REP_LAYERSTYLE_TYPE, "primary");
-                        representation.updateContentFromPath(stylePath, false, done);
-                    })
+                    var representation = newElement.createRepresentation(REP_LAYERSTYLE_TYPE, "primary");
+                    representation.updateContentFromPath(stylePath, false, done);
+                })
                     .then(function () {
```

Comment:
This looks wrong to me. Maybe we should remove the indents of the below code block as well.